implementation/src/main/java/io/smallrye/health/SmallRyeHealthReporter.java

```diff
@@ -158,17 +117,22 @@ private JsonObject jsonObject(HealthCheck check) {
         try {
             return jsonObject(check.call());
         } catch (RuntimeException e) {
+            // Print Stacktrace to server log so an error is not just in Health Check response
+            e.printStackTrace();
```

Comment:
`printStackTrace` is always a bad way to log an error :-)

Reply:
@Ladicek pushed a commit to use logging